module/VuFind/src/VuFind/Db/Entity/SearchEntityInterface.php

```diff
@@ -40,4 +42,170 @@
  */
 interface SearchEntityInterface extends EntityInterface
 {
+    /**
+     * Get identifier (returns null for an uninitialized or non-persisted object).
+     *
+     * @return ?int
+     */
+    public function getId(): ?int;
+
+    /**
+     * Get user.
+     *
+     * @return int
+     */
+    public function getUserId(): int;
+
+    /**
+     * Set user.
+     *
+     * @param int $user User
+     *
+     * @return SearchEntityInterface
+     */
+    public function setUserId(int $user): SearchEntityInterface;
+
+    /**
+     * Get session.
+     *
+     * @return ?string
+     */
+    public function getSessionId(): ?string;
+
+    /**
+     * Set session.
+     *
+     * @param ?string $session_id Session id
+     *
+     * @return SearchEntityInterface
+     */
+    public function setSessionId(?string $session_id): SearchEntityInterface;
+
+    /**
+     * Get created.
```

Comment:
I'd add "date" to the end of this sentence for consistency with the setter below.